### ISSUE TITLE
ESLint does not produce correct errors is there are some specific configs in file

### ISSUE THREAD

**Issue by roadhump** [eslint/eslint#819]
If we have two files in folder, `eslint .` will not produce `quotes` error for the second one.

``` js
// bar.js
/*eslint quotes:0 */
var b = '1';

// foo.js
var f = '1';
```
